### PR TITLE
license

### PR DESCRIPTION
**Pull request by alegmarra**

### CODE REVIEW

File: mule-sdk/v/4.0/license.adoc [hunk 1]
```diff
@@ -0,0 +1,142 @@
+= Modules Licensing
+
+Modules written with the SDK have three possible kind of licensing, each setting different requirements
```
Comment (estebanwasinger) [inline mule-sdk/v/4.0/license.adoc]: `kind` -> `kinds`

Comment (estebanwasinger) [reply mule-sdk/v/4.0/license.adoc]: `each setting` -> `each set` ?

File: mule-sdk/v/4.0/license.adoc [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+@Extension(name = "enterprise")
+@RequiresEnterpriseLicense
+public class EnterpriseConnector {
+
+}
+----
+
+By default, `RequiresEnterpriseLicense` forces the user to have a full MuleSoft Enterprise License,
+but you can also declare the Module to allow an evaluation license to be used in for the Runtime.
+This can be configured using the `allowEvaluationLicense` flag in the `RequiresEnterpriseLicense` annotation:
```
Comment (estebanwasinger) [inline mule-sdk/v/4.0/license.adoc]: `Runtime. This can be configured` -> `Runtime, this can be configured` ?

File: mule-sdk/v/4.0/license.adoc [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+@Extension(name = "enterprise")
+@RequiresEnterpriseLicense(allowEvaluationLicense = true)
+public class EnterpriseConnector {
+
+}
+----
+
+== Custom Module License
+
+MuleSoft Certified Connectors, can request to MuleSoft a _certified license generation key_ that allows them to
```
Comment (estebanwasinger) [inline mule-sdk/v/4.0/license.adoc]: `Connectors, can request` -> `Connectors can request`

File: mule-sdk/v/4.0/license.adoc [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+.. *Company Name* (also known as "Provider Name"), *Contact Email*, *Contact Message*, *Connector Name*
+. Request the key and keystore from MuleSoft which are:
+.. The public `.key` file that you store in the connector
+.. The `.jks` master keystore is for you as the partner to generate licenses with.
+
+== Adjusting the Connector Code and Storing the Public Key
+
+After obtaining the key and keystore:
+
+. The partner adjusts the connector project code by applying the `@RequiresEntitlement` annotation, and entering values for all requested attributes.
```
Comment (estebanwasinger) [inline mule-sdk/v/4.0/license.adoc]: `annotation, and entering` -> `annotation and entering`

File: mule-sdk/v/4.0/license.adoc (hunk 2 above)
Comment (alegmarra) [reply mule-sdk/v/4.0/license.adoc]: nope

Comment (estebanwasinger) [reply mule-sdk/v/4.0/license.adoc]: Technically you are loosing the subject

**Pull request merged**